javascript/src/seedsource/containers/Map.jsx

```diff
@@ -610,13 +579,43 @@ class Map extends React.Component {
         }
     }
 
-    updateVectorLayers(layers) {
-        let numLayersToAdd = layers.length - this.displayedVectorLayers.length
+    updateRasterLayers(layers) {
+        let numLayersToAdd = layers.length - this.displayedRasterLayers.length
+
+        if (numLayersToAdd > 0) {
+            this.displayedRasterLayers.push(...Array(numLayersToAdd).fill().map(
```

Comment:
The `"placeholder"` bit has been causing tile requests for "palceholder" (which obviously fail). It's not a huge problem (it still works from the user's POV), but let's think about how to avoid this.

How about figuring out the index when adding the layers and using the action URL instead?

```javascript
this.displayedRasterLayers.push(...Array(numLayersToAdd).fill().map((_, index) => {
    let layer = layers[this.displayedRasterLayer.length+index]
    let url = getLayerUrl(layer, serviceId, objective, climate, region)
    return L.tileLayer(`/tiles/${url}/{z}/{x}/{y}.png`, {zIndex: 1, opacity: 1}).addTo(this.map)
}))
```